system-int-test/src/cmr/system_int_test/system.clj

```diff
@@ -41,7 +41,7 @@
              ;; A map of the components (echo, elastic, db, and message queue) to whether they are
              ;; in-memory or external
              :component-type-map component-type-map}]
-    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :urs])))
+    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :urs :ingest])))
```

Comment:
This is so that we can hit ingest through the transmit-lib. We have always been hitting ingest directly through `clj-http.client/request`. I added this to make use of the defmacros defined in `cmr.transmit.http-helper`.